scripts/build-archive/build-archive.go

```diff
@@ -44,21 +46,38 @@ func createArchive(pluginName string) error {
 
 	// Then let's create the archive with the folder previously created
 	archiveName := fmt.Sprintf("%s-%s.tar.gz", manifest.ID, manifest.Metadata.BuildInfo.Version)
-	if execErr := exec.Command("tar", "-czvf", path.Join(pluginName, archiveName), "-C", pluginName, pluginName).Run(); execErr != nil {
+	args := []string{"-czvf", path.Join(pluginName, archiveName), "-C", pluginName, pluginName}
+
+	// Remove the copyfile metadata on macos
+	if runtime.GOOS == "darwin" {
+		args = append([]string{"--disable-copyfile"}, args...)
+	}
+
+	if execErr := exec.Command("tar", args...).Run(); execErr != nil {
 		return execErr
 	}
 
+	if createGroupArchive {
+		if execErr := exec.Command("cp", path.Join(pluginName, archiveName), path.Join("./plugins-archive", archiveName)).Run(); execErr != nil {
```

Comment:
Maybe I forgot something, but here I don't get where the binary `plugins-archive` is coming from. I can admit I don't even get why we would need that

Reply:
It was just for convenience while developing locally to replace it in the Perses side. But we can remove it once we can load a development version of the plugin files from the `/plugins/` endpoint in Perses.

Reply:
ah yeah makes sense. We will need to provide something quickly, it will be super annoying otherwise to develop a plugin